docs/reST/ref/math.rst

```diff
@@ -195,6 +195,36 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.distance_squared_to ##
 
+   .. method:: move_towards
+
+      | :sl:`returns a vector that moves toward to the given vector at the given distance.`
+      | :sg:`move_towards(Vector2, float) -> Vector2`
+
+      Similar to ``lerp()``, this returns a Vector which moves self to the
+      given Vector at a given distance. The first parameter determines the target
+      Vector, while the second parameter determines the distance. The distance value
+      must be greater than 0, but if the distance value is equal to 0, it will return
+      self.
+
+      .. versionadded:: 2.1.2
+
+      .. ## Vector2.move_towards ##
+
+   .. method:: move_towards_ip
+
+      | :sl:`returns a vector that moves toward to the given vector at the given distance.`
+      | :sg:`move_towards_ip(Vector2, float) -> Vector2`
```

Comment:
`_ip` does not return a `Vector2`, it returns `None` and modifies the vector inplace.

The docs below this line need updating too